src/redux/actions/auth/authActions.js

```diff
@@ -19,10 +19,11 @@ const loginAction = ({ username, password }) => (dispatch) => {
     localStorage.setItem('username', `${username}`);
     return dispatch(login(response));
   }).catch((error) => {
-    if (error.response) {
-      return dispatch(loginFailed(error.response.data.detail));
+    const offlineError = 'Network Error. Please try again later!';
```

Comment:
Don't we already have this here? https://github.com/openmrs/openmrs-ocl-client/search?q=showNetworkError&unscoped_q=showNetworkError

Reply:
Does that mean we are free to duplicate the same thing across multiple files?

Reply:
Maybe am getting it wrong, so should I import it from the other file?

Reply:
I do not want to dictate to you in regards to how you should address it. Am simply raising the important software engineering principle of not duplicating code. Are you aware of it?

Reply:
Yeah, am aware of it. So I bet importing and reusing it serves that principle well.

Reply:
I have reused it.